Traceback (most recent call last):
  File "./frankenstein.py", line 433, in <module>
    frank.runlive()
  File "./frankenstein.py", line 393, in runlive
    self.check()
  File "./frankenstein.py", line 224, in check
    lastbar = getFeed(self.symbol, 2, self.interval)
  File "./frankenstein.py", line 60, in getFeed
    data = dbhist.get_hist(symbol, interval, lookback).sort_index(ascending=True)
  File "E:\Frankenstein\scripts\iqfeed\dbhist.py", line 55, in get_hist
    s = socket.create_connection((READER_HOSTNAME, READER_PORT))
  File "d:\anaconda2\lib\socket.py", line 575, in create_connection
    raise err
socket.error: [Errno 10061] No connection could be made because the target machine actively refused it
